core/src/toga/app.py

```diff
@@ -349,6 +348,27 @@ def __init__(
     def _create_impl(self):
         return self.factory.App(interface=self)
 
+    @property
+    def paths(self):
+        """
+        Paths for known safe system locations. Using paths makes sure you have a consistent and known location to store
+        files for your app. Also, some platforms do not allow arbitrary file access to any location on disk. On Android,
+        there are many limitations when it comes to accessing file locations and the apis you can use to read from those
+        locations.
+
+        PROPERTIES:
```

Comment:
It would be better to use native ReST markup, rather than capitalisation and indentation. This is effectively a subheading with bullet point descriptions.

It probably also warrants a description of why a property (`paths`) has properties.